atom/common/linux/application_info.cc

```diff
@@ -2,18 +2,74 @@
 // Use of this source code is governed by the MIT license that can be
 // found in the LICENSE file.
 
+#include "brightray/common/application_info.h"
+
+#include <gio/gdesktopappinfo.h>
+#include <gio/gio.h>
+
+#include <memory>
 #include <string>
 
 #include "atom/common/atom_version.h"
+#include "base/environment.h"
+#include "base/logging.h"
+#include "chrome/browser/ui/libgtkui/gtk_util.h"
+
+namespace {
+
+GDesktopAppInfo* get_desktop_app_info() {
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
+  const std::string desktop_id = libgtkui::GetDesktopName(env.get());
```

Comment:
Doesn't it default to `chromium-browser.desktop` https://cs.chromium.org/chromium/src/chrome/browser/ui/libgtkui/gtk_util.cc?type=cs&q=GetDesktopname&sq=package:chromium&l=132 when the `CHROME_DESKTOP` env isn't set, should we avoid this default value ?

Reply:
I agree with both these points & will address them in the `linux-appname` branch

Reply:
Addressed in https://github.com/electron/electron/pull/11957/commits/38dd5cee68fd5587f10de1e40b2d9291b8d8b3ba and https://github.com/electron/electron/pull/11957/commits/c8d2487747a0ffcd579c8f696a25e61beab29f00